staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -78,11 +77,17 @@ var (
 		# (requires the EphemeralContainers feature to be enabled in the cluster)
 		kubectl alpha debug --image=myproj/debug-tools -c debugger mypod
 
-		# Create a debug container as a copy of the original Pod and attach to it
+		# Create a copy of mypod adding a debug container and attach to it
 		kubectl alpha debug mypod -it --image=busybox --copy-to=my-debugger
 
-		# Create a copy of mypod named my-debugger with my-container's image changed to busybox
-		kubectl alpha debug mypod --image=busybox --container=my-container --copy-to=my-debugger -- sleep 1d
+		# Create a copy of mypod changing the command of mycontainer
+		kubectl alpha debug mypod -it --copy-to=my-debugger --container=mycontainer -- sh
+
+		# Create a copy of mypod changing all container images to busybox
+		kubectl alpha debug mypod --copy-to=my-debugger --set-image=*=busybox
+
+		# Create a copy of mypod adding a debug container and changing container images
+		kubectl alpha debug mypod -it --copy-to=my-debugger --image=debian --set-image=app=app:debug,sidecar=sidecar:debug
 
```

Comment:
@soltysh When implementing image mutation I realized it might be more intuitive to have a separate flag, using `--image` for adding a new debug container and `--set-image` for changing the images of existing containers. With this syntax it's possible to perform both operations in a single command, but mainly I just think the syntax is clearer.

Could you have a look at these examples and let me know what you think? For reference, we previously discussed [overloading the `--image` flag for both uses](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/1441-kubectl-debug#modify-application-image-by-copy)